Modules/Microsoft365DSC/DSCResources/MSFT_O365SearchAndIntelligenceConfigurations/settings.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+    "resourceName": "O365SearchAndIntelligenceConfigurations",
+    "description": "",
+    "permissions": {
+        "graph": {
+            "delegated": {
+                "read": [],
+                "update": []
+            },
+            "application": {
+                "read": [],
```

Comment:
Looks strange that there are Update permissions, but no Read permissions. Is this correct?